### ISSUE TITLE
Write an abbreviated README for the packaged distribution

### ISSUE THREAD

**Issue by vdauwera** [broadinstitute/gatk#3199]
Readme packaged with the jars should be more user-friendly, omit dev instructions such as how to compile (since the package by definition does not include source code) and start right away with quickstart instructions:

- how to run 
- where to find docs
- where to get help

Could be largely lifted from http://gatkforums.broadinstitute.org/gatk/discussion/9881/howto-get-started-with-gatk4-beta

**Comment by droazen** [comment]
Agree that a separate README for the binary distribution is a good idea, but the document linked to above lacks basic instructions on things like running on a cluster and setting up GCS authentication. I think the doc should be based on the repo README instead with some sections omitted. It would actually be best if it could be generated automatically somehow from the repo README, so that we don't have to edit two documents whenever we make a change.

**Comment by vdauwera** [comment]
I agree that would be even better. Can we markup the README somehow and get the relevant sections spliced accordingly at build time?

**Comment by droazen** [comment]
Possibly -- do you happen to know of a good way to do this, @vdauwera ?

**Comment by vdauwera** [comment]
Not really... after further thought I think it would be cleaner to break down the main readme into two docs -- one quickstart doc for end users with usage instructions etc that we can include in the distro, and one for developers with compilation instructions etc that is repo only. I know you didn't want to move anything to the wiki in order to keep everything versioned, which I agree makes sense -- so this would preserve that. 

Either one could be the "main" README, and each would have a preface pointing at the other, or we could have a very short master README pointing to both.

**Comment by cmnbroad** [comment]
It wouldn't be that hard to write a build step that does the splicing with m4.

**Comment by droazen** [comment]
Hmm...is there a way to "inline" one markdown file in another?

**Comment by vdauwera** [comment]
not that I'm aware

**Comment by droazen** [comment]
I'd prefer a solution that keeps the main README intact if at all possible, since it's very useful to be able to tell people "go to the README and search for X" in response to questions. Recommend that we look into whether there are any markdown features we could use to help us out here.

**Comment by vdauwera** [comment]
Can I ask what kind of questions you get and from what audience, that you answer this way? Anything usage related should hopefully migrate to the end-user website as we move GATK4 to being our prime supported version. 

For the rest I should hope people are able to read a line at the top of a readme document that says "for additional information about X click here" ...

To be blunt, I just want to make sure we don't over-complicate our lives only to enable a few people to be lazy.

**Comment by droazen** [comment]
Questions from developers, mainly, about things like "my IntelliJ project is broken" and "GCS authentication not working" -- you get the picture :)

**Comment by droazen** [comment]
Might be easier to discuss this ticket face-to-face to try to brainstorm about what the relationship should be between the docs targeted at users of the git repository vs. docs targeted at GATK end-users. Feel free to put a meeting on my calendar @vdauwera

**Comment by vdauwera** [comment]
I agree :) 

Meeting in person may be hard (I leave for the UK next week) but we can probably swing a hangout. No huge rush anyway.
